### ISSUE TITLE
documentation for default_networks

### ISSUE THREAD

**Issue by StanleyShen** [cloudfoundry/cf-release#894]
In cf generated bosh-lite manifest file, I notice some sections like "default_networks", what does it mean?
I didn't find any document on bosh http://bosh.io/docs/deployment-manifest.html

```
- default_networks:
  - name: cf1
    static_ips: null
  instances: 1
  name: ha_proxy_z1
  networks:
  - name: cf1
    static_ips:
    - 10.244.0.34
  properties:
    ha_proxy:

```

**Comment by Amit-PivotalLabs** [comment]
Hi @StanleyShen 

What you have there is (part of) an array of jobs, and you're looking at the first element in that array.  That element has `name: ha_proxy_z1`, so it's the job definition for the HA Proxy in zone 1.  spiff arranges keys alphabetically, so you see `default_networks` at the top, but the identifying property is the name.

You still may ask, within the job definition for `ha_proxy_z1`, what is `default_networks`.  You'll notice that this is not mentioned in the bosh.io docs, and indeed BOSH will basically ignore the `default_networks` data.  It's only there for the sake of spiff.  In the templates, you have:

https://github.com/cloudfoundry/cf-release/blob/master/templates/cf-jobs.yml#L237-L240

```
  - name: ha_proxy_z1
    ...
    default_networks:
      - name: cf1
        static_ips: ~
    networks: (( merge || default_networks ))
    ...
```

This basically says that the `networks` section of the job will either equal whatever is provided by the stubs (that's the `merge` part), or defaults to the value in `default_networks`.

**Comment by StanleyShen** [comment]
yes, for this part, I understand it.
But as you see, the "default_networks" I pasted here is at the same level of jobs. (I saw it in one old cf.yml locally, but it seems not there in latest generated cf.yml)
Is it the same meaning?

- instances: 0
  name: consul_z2
  networks:
  - name: cf2
    static_ips: []
  persistent_disk: 1024
  properties:
    consul:
      agent:
        mode: server
    metron_agent:
      zone: z2
  resource_pool: small_z2
  templates:
  - name: consul_agent
    release: cf
  - name: metron_agent
    release: cf
  update:
    max_in_flight: 1
    serial: true
- default_networks:
  - name: cf1
    static_ips: null
  instances: 1
  name: ha_proxy_z1
  networks:
  - name: cf1
    static_ips:
    - 10.244.0.34
  properties:
    ha_proxy:
